plugin-server/src/types.ts

```diff
@@ -288,6 +289,8 @@ export interface Plugin {
     is_stateless?: boolean
     public_jobs?: Record<string, PluginPublicJobPayload>
     log_level?: PluginLogLevel
+    latest_tag?: string
```

Comment:
What are these? Do we require them in the plugin server?

Reply:
I don't think we do, but this is returned and I'm strongly in favor of types being fully representative of the data structures. Though I can also just explicitly omit this in `getPluginRows()`. What do you think?

Reply:
Let's explicitly not fetch data not being used by plugin server.